contracts/event/MultipleResultsEvent.sol

```diff
@@ -562,4 +519,139 @@ contract MultipleResultsEvent is NRC223Receiver, Ownable {
+            for (uint8 j = 0; j < _numOfResults; j++) {
+                uint total = _eventRounds[i].balances[j].total;
+                if (j == _currentResultIndex) {
+                    voteRoundsWinnersTotal = voteRoundsWinnersTotal.add(total);
+                } else {
+                    voteRoundsLosersTotal = voteRoundsLosersTotal.add(total);
+                }
+            }
+        }
+
+        // Calculate user's winning amount for vote rounds
+        uint voteRoundsWinningAmt;
+        if (myWinningVotes > 0 && voteRoundsWinnersTotal > 0) {
+            voteRoundsWinningAmt =
+                betRoundLosersTotal
+                .mul(_arbitrationRewardPercentage)
+                .div(maxPercent)
+                .add(voteRoundsLosersTotal)
+                .mul(myWinningVotes)
+                .div(voteRoundsWinnersTotal);
+        }
+
+        return myWinningBets
+            .add(myWinningVotes)
+            .add(betRoundWinningAmt)
+            .add(voteRoundsWinningAmt);
+    }
+
+    /// @notice Calculates the tokens returned for an Invalid final result.
+    /// @param better Address to calculate winnings for.
+    /// @return Amount of tokens that will be returned.
+    function calculateInvalidWinnings(
+        address better)
+        private
+        view
+        returns (uint)
+    {
+        // Calculate user's winning bets
+        uint myWinningBets;
+        uint myWinningVotes;
+        for (uint8 i = 0; i <= _currentRound; i++) {
+            uint bets = _eventRounds[i].balances[_currentResultIndex].bets[better];
+            if (i == 0) {
+                myWinningBets = myWinningBets.add(bets);
+            } else {
+                myWinningVotes = myWinningVotes.add(bets);
+            }
+        }
+
+        // Calculate user's losing bets
+        uint myLosingBets;
+        for (uint8 i = 0; i < _numOfResults; i++) {
+            if (i != _currentResultIndex) {
+                myLosingBets = myLosingBets.add(_eventRounds[0].balances[i].bets[better]);
+            }
+        }
+
+        // Calculate all vote rounds totals
+        uint voteRoundsWinnersTotal;
+        uint voteRoundsLosersTotal;
+        for (uint8 i = 1; i <= _currentRound; i++) {
+            for (uint8 j = 0; j < _numOfResults; j++) {
+                uint total = _eventRounds[i].balances[j].total;
+                if (j == _currentResultIndex) {
+                    voteRoundsWinnersTotal = voteRoundsWinnersTotal.add(total);
+                } else {
+                    voteRoundsLosersTotal = voteRoundsLosersTotal.add(total);
+                }
+            }
+        }
+
+        // Calculate user's winning amount for vote rounds
+        uint voteRoundsWinningAmt;
+        if (myWinningVotes > 0 && voteRoundsWinnersTotal > 0) {
+            voteRoundsWinningAmt =
+                voteRoundsLosersTotal
+                .mul(myWinningVotes)
+                .div(voteRoundsWinnersTotal);
+        }
```

Comment:
else {
voteRoundsWinningAmt = 0
}

Reply:
not necessary. default values are always the zero value.